app/controllers/application_controller.rb

```diff
@@ -3,9 +3,27 @@
 class ApplicationController < ActionController::API
   include ActionController::MimeResponds
 
+  rescue_from ActiveRecord::RecordNotFound, with: :not_found
+  rescue_from Moab::ObjectNotFoundException, with: :not_found
+  rescue_from InvalidSuriSyntax, with: :bad_request
```

Comment:
my impression from prior rails work is that `rescue_from` with handler methods is more idiomatic than rescuing inline in the child controller.  it certainly seems more re-usable to me.